codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/StubsGeneratorBase.java

```diff
@@ -353,16 +352,6 @@ protected void renderMemberDefaults(Shape s) {
         writer.closeBlock("}");
     }
 
-    // TODO: should probably be in Rest Base
```

Comment:
I'm not quite convinced this should be in Rest Base (though this is HTTP specific, I believe it applies to every protocol we know right now.  But I guess this is where our only 2 levels of base classes falls short.  But since this would need to be duplicated in alot of the v4 protocols, I'd potentially lean towards leaving it here but making the todo more descriptive)

Reply:
It's used in restxml, restjson, fakeprotocol, and railsjson, all of which inherit from rest base. I think it belongs in an http specific base but rest base I think is a better home for it now. I don't believe this applies to other protocols like json 1.0/1.1, ec2, and query.